javatests/arcs/core/storage/ReferenceModeStoreTest.kt

```diff
@@ -700,6 +700,31 @@ class ReferenceModeStoreTest {
     assertThat(store.backingStore.stores.size).isEqualTo(0)
   }
 
+  @Test
+  fun close_closesBackingAndContainerStores() = runBlockingTest {
+    val activeStore = createSingletonReferenceModeStore()
+    val actor = activeStore.crdtKey
+    val bob = createPersonEntity("an-id", "bob", 42)
+
+    // Set singleton to Bob.
+    val updateOp = RefModeStoreOp.SingletonUpdate(actor, VersionMap(actor to 1), bob)
+    activeStore.onProxyMessage(ProxyMessage.Operations(listOf(updateOp), id = 1))
+
+    assertThat(activeStore.containerStore.closed).isFalse()
+    assertThat(activeStore.backingStore.stores.size).isEqualTo(1)
+    activeStore.backingStore.stores.values.forEach { storeRecord ->
+      assertThat(storeRecord.store.closed).isFalse()
+    }
```

Comment:
Since there's only one entry, there's no need for the loop, and you can just use `.single()` to get the single value